data/templates/styles.html

```diff
@@ -23,6 +23,9 @@
     font-size: 0.9em;
     padding: 1em;
   }
+  h1 {
+    font-size: 1.8em;
```

Comment:
What is the default for this - or does it vary with the browser?
Also what is the default for h2, h3, etc.?

Reply:
My specific worry is that if you set the h1 font-size to 1.8, it will be indistinguishable from h2.  Maybe we should explicitly set all hN font sizes, using some kind of harmonious scheme?

Reply:
By the way, my own aesthetics prefer much smaller headings (more like what you get in print articles).  However, then you have some difficulty distinguishing 5 or 6 heading levels.  It can perhaps still be done by not using bold fonts for all of them.

Reply:
Different browsers [used](https://stackoverflow.com/a/6140504/214446) to have different values for this, but nowadays this is all [standardized](http://zuga.net/articles/html-heading-elements/). I just tested Chrome, Firefox and Safari on macOS and all set the `h1` to `2em`. With 1em usually being 16px, this would be 32px, but we define 1em on desktop-sized screens with `html { font-size: 20px}`, so in this stylesheet it's 40px.

> Maybe we should explicitly set all hN font sizes, using some kind of harmonious scheme?

yes, that's what people used to do (and had to do due to browser differences). My idea with this stylesheet was to rely on browser defaults as much as possible, to keep it small. And I think it worked out pretty well generally... except that the h1 is too big on mobile :P But arguably it's not too bad and we can also just leave it...

> My specific worry is that if you set the h1 font-size to 1.8, it will be indistinguishable from h2

That would be bad indeed, but I tested it and it works out like this: We have:

```
@media (max-width: 600px) {
  body {
    font-size: 0.9em;
```
so on mobile, 1em is actually `20px * 0.9 = 18px`. Thus:

h2 is 1.5em (by browser default) -> 27px.
If we set h1 to 1.8em -> 32.4px.

(you can confirm these pixel values by looking at the "calculated" tab in the browser's dev tools when having an h1/h2 element selected)

Reply:
>  then you have some difficulty distinguishing 5 or 6 heading levels. It can perhaps still be done by not using bold fonts for all of them. 

we already do that. h4, h5, h6 and paragraph text all are 20px, but:

<img width="336" alt="Screenshot 2022-01-15 at 09 15 48" src="https://user-images.githubusercontent.com/157497/149614900-933a250f-9f62-42c4-9dc5-e671674b1686.png">

Reply:
OK, this sounds reasonable then!

Reply:
great, I updated the tests...